src/components/DataViewer/DataViewer.tsx

```diff
@@ -15,45 +17,70 @@ const StyledModal = styled(Modal)`
     left: 50%;
     transform: translate(-50%, -50%);
     border: 0;
-    border-radius: 5px;
+    border-radius: 2em;
     outline: none;
     padding: 0;
     width: 80%;
     max-height: 80%;
-    background: rgb(255, 255, 255);
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
     box-shadow: rgb(200, 200, 200) 0 2px 10px;
     overflow: hidden;
+    max-width: 1000px;
 `;
-const Header = styled.h3`
+
+const Header = styled.div`
     display: flex;
     justify-content: space-between;
     align-items: center;
-    background-color: #eeeeee;
-    padding: 0.5rem;
-    margin-top: 0;
+    color: ${(props) => props.theme.secondaryForegroundColor};
+    background-color: ${(props) => props.theme.secondaryBackgroundColor}A0;
+    font-size: 1.5em;
+    font-weight: bolder;
+    padding: 0.75em 1em;
 `;
+
 const Content = styled.div`
-    overflow: scroll;
-    margin: 1.5rem;
-    margin-top: 1rem;
+    overflow: auto;
+    margin: 1rem 1.5rem 1.5rem;
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
 `;
+
 const Key = styled.div`
-    color: grey;
-    font-size: small;
+    color: ${(props) => props.theme.foregroundColor};
```

Comment:
can this be `secondaryForegroundColor` - it's not really that great to mix and match colours! Pick on type (secondary) and stick with it :)

Reply:
Refactored

Reply:
This is an issue of VFB-16 and we have left a note on this for the person handling VFB-16 to work  on the color palette

Reply:
I agree that most of the time we shouldn't leave comments in code, but in this case it's referring to another ticket which currently is in 'todo', meaning someone else will pick it up. It has a ticket (VFB-16) and this is a good way to link the Jira ticket with code, and the ticket shouldn't be finished until this is done!

I do think this is a non-trivial amount of work that _should_ be done in the separate ticket - if it was tiny then it should be done, but this is 'create a whole new palette of colours', which fits 16 better than 3! There is also a 'quick fix' here - using 40 as opacity, which is 'fine for main' but should be fixed as 'tech debt' - which is why they've left the comment.